sdk/resourcemanager/azure-resourcemanager-appplatform/README.md

```diff
@@ -22,10 +23,46 @@ For documentation on how to use this package, please see [Azure Management Libra
 ```
 [//]: # ({x-version-update-end})
 
+### Include the recommended packages
```

Comment:
pls keep your script for future maintaince. :) otherwise we can put a link of main readme.md auth part here.

Reply:
Sure.

The link to main readme auth is already here in line 56. "See [Authentication][authenticate] for more options."